app/src/main/java/com/concordium/wallet/ui/common/BackendErrorHandler.kt

```diff
@@ -44,6 +44,9 @@ object BackendErrorHandler {
             0 -> {
                 R.string.app_error_backend_internal_server
             }
+            1->{
```

Comment:
```suggestion
            1 -> {
```

Reply:
Please fix the formatting as suggested by Michael.